### PR TITLE
common: add override in common and misc

### PR DESCRIPTION
**Pull request by Liuchang0812**
Fixes: http://tracker.ceph.com/issues/18922
Signed-off-by: liuchang0812 <liuchang0812@gmail.com>

### CODE REVIEW

**Comment by liewegas** [comment]
There are several .swn files in here

**Comment by Liuchang0812** [comment]
sorry, i will fix it tomorrow morning

**Comment by yuriw** [comment]
test this please

**Comment by Liuchang0812** [comment]
I have updated it. Waiting test result.

File: src/test/system/rados_list_parallel.cc [hunk 1]
```diff
@@ -52,11 +52,11 @@ class RadosDeleteObjectsR : public SysTestRunnable
   {
   }
 
-  ~RadosDeleteObjectsR()
+  ~RadosDeleteObjectsR()
```
Comment (tchaikov) [inline src/test/system/rados_list_parallel.cc]: @Liuchang0812 trailing space.

File: src/test/system/rados_list_parallel.cc [hunk 2]
```diff
@@ -137,11 +137,11 @@ class RadosAddObjectsR : public SysTestRunnable
   {
   }
 
-  ~RadosAddObjectsR()
+  ~RadosAddObjectsR()
```
Comment (tchaikov) [inline src/test/system/rados_list_parallel.cc]: ditto

File: src/test/system/rados_list_parallel.cc (hunk 1 above)
Comment (Liuchang0812) [inline src/test/system/rados_list_parallel.cc]: Good catch! I will remove it

Comment (Liuchang0812) [inline src/test/system/rados_list_parallel.cc]: Done

File: src/test/system/rados_list_parallel.cc (hunk 2 above)
Comment (Liuchang0812) [inline src/test/system/rados_list_parallel.cc]: Done.

there are a lot of tailing spaces in code base.

```
➜  ceph git:(cleanup-common) ✗ grep -n "\ $" `git diff --name-only HEAD^` | wc -l
grep: src/boost: Is a directory
498

➜  ceph git:(cleanup-common) ✗ grep -nr "\ $" src | wc -l                        
304750
```

Comment (tchaikov) [inline src/test/system/rados_list_parallel.cc]: @Liuchang0812 just don't want to introduce more in new commits.

Comment (Liuchang0812) [inline src/test/system/rados_list_parallel.cc]: @tchaikov roger that, I just fixed above 2 lines.

**Pull request merged**